macros/utils/data_types/cast_column.sql

```diff
@@ -6,6 +6,10 @@
     cast({{ column }} as {{ elementary.type_float() }})
 {%- endmacro -%}
 
+{%- macro cast_as_numeric(column) -%}
+    cast({{ column }} as {{ elementary.type_numeric() }})
+{%- endmacro -%}
```

Comment:
Can add context on why casting to `type_numeric` was needed?

Reply:
The `round()` function of Postgres expects this data type as its first argument.

```sql
Database Error
  function round(double precision, integer) does not exist
  LINE 16:                 else round(execution_time, 1)
                                ^
  HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
```

Reply:
👍